assets/scss/components/_forms.scss

```diff
@@ -151,7 +151,7 @@ Faux inputs visually mimic actual input elements.
 }
 
 $toggleSize: 20px;
-$C_toggleHighlight: $C_lightPurple; // replace $C_lightPurple with $C_blue once we start to use `swarm-constants`
+$C_toggleHighlight: $C_blue;
```

Comment:
Not really part of this PR, but I figured I'd update SASS vars after upgrading to a version of `swarm-sasstools` that pulls from `swarm-constants`